### ISSUE TITLE
How could I reproduce your result (31.76dB in PSNR) in Table 2?

### ISSUE THREAD

**Issue by pp00704831** [jibo27/MemDeblur#6]
Hello,

I tested the model with pre-trained weights on the original GOPRO test set by setting 'test_frames' and 'test_mem_length' to 5.
However, I only obtained 31.36dB in PSNR.  How could I obtain the 31.76dB result as shown in Table2? 
Besides, do I need to change the number of blocks or others to obtain the same GMACs (344.49 or 325.48) result in Table 2?

Thank you!
